content/en/agent/kubernetes/daemonset_setup.md

```diff
@@ -194,6 +204,72 @@ For Agent v6.11+, the Datadog Agent can auto-detect the Kubernetes cluster name
 
 ## Enable capabilities
 
+### APM and Distributed Tracing
+
+To enable APM by allowing incoming data from port 8126, uncomment the following portion in your YAML file:
+
+```yaml
+- containerPort: 8126
+  hostPort: 8126
+  name: traceport
+  protocol: TCP
+```
+
+Make sure that `DD_APM_ENABLED` is set to true:
+```yaml
+- {name: DD_APM_ENABLED, value: "true" }
+```
+
+**Important:** in order for the tracing libraries to find the agent, you need to add the following enviornement variable to every deployment file that belongs to an app you'd like to instrument:
+
+<pre>
+apiVersion: apps/v1
+kind: Deployment
+...
+    spec:
+      containers:
+      - name: <CONTAINER_NAME>
+        image: <CONTAINER_IMAGE>/<TAG>
+      <b>env:
```

Comment:
the formatting is off

<img width="647" alt="image" src="https://user-images.githubusercontent.com/5287287/76443921-74593f80-63c3-11ea-9a2b-c8a5e7da2103.png">